android/app/build.gradle

```diff
@@ -127,14 +127,14 @@ def jscFlavor = 'org.webkit:android-jsc:+'
  */
 def enableHermes = project.ext.react.get("enableHermes", false);
 
+/**
+ * Architectures to build native code for in debug.
```

Comment:
This is minor, but could you adjust this message slightly? It's not 100% clear what 'for in debug means'.

> Required in order to build native code for debug builds

Reply:
Yeah, this was copy/pasted from the upgrade guide. I can adjust the message here.